perfkitbenchmarker/windows_benchmarks/geekbench_benchmark.py

```diff
@@ -0,0 +1,164 @@
+# Define the name of the benchmark as a string constant.
+BENCHMARK_NAME = 'geekbench'
+
+# Define the configuration for the benchmark.
+# This includes VM groups and any flags specific to this benchmark.
+BENCHMARK_CONFIG = """
+geekbench:
+  description: >
+    Runs Geekbench 6 to evaluate system performance across CPU and GPU on
+    Linux or Windows platforms.
+  vm_groups:
+    default:
+      vm_spec: *default_single_core
+"""
+
+# Import necessary modules from PKB
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import sample
+
+def GetConfig(user_config):
+    """
+    Returns the configuration for the benchmark.
+    """
+    return configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
+
+def Prepare(benchmark_spec):
+    """
+    Sets up the environment on the VM for the benchmark.
+    """
+    pass 
```

Comment:
Curious why you are not calling install here when you already implemented the installation code. 
i.e. 
vm = benchmark_spec.vms[0]
vm.install('geekbench')

Reply:
Got it!